.travis.yml

```diff
@@ -42,7 +42,8 @@ env:
     - TRAVIS_FLAVOR=apache FLAVOR_VERSION=2.4.12
     - TRAVIS_FLAVOR=cassandra FLAVOR_VERSION=2.0.17
     - TRAVIS_FLAVOR=cassandra FLAVOR_VERSION=2.1.14
-    - TRAVIS_FLAVOR=couch FLAVOR_VERSION=1.6.1
+    - TRAVIS_FLAVOR=couch FLAVOR_VERSION=couchdb:1.6.1
+    - TRAVIS_FLAVOR=couch FLAVOR_VERSION=klaemo/couchdb:2.0-dev
```

Comment:
Generally, we prefer to keep the logic of which docker image to pull in the rakefile instead of here. This might be a better way to do it in some ways but the seeing which test is failing at a glance is easier when the logic is separated.

We had a conversation about this internally, because there is some appeal to having the repo configurable by env var. We might move to something like that, but for now, just move the logic into the rakefile. We do this in a few places, you can see with elastic search: we have [a lot of versions of it](https://github.com/DataDog/integrations-core/blob/master/.travis.yml#L49-L59), and we separate out where they come from [in the rakefile](https://github.com/DataDog/integrations-core/blob/master/elastic/ci/elastic.rake#L25-L30) .